.github/workflows/cron.yml

```diff
@@ -0,0 +1,46 @@
+name: Cron
+on:
+    schedule:
+        - cron:  '0 23 * * *'
+jobs:
+  js-routing-dump:
+    runs-on: ubuntu-latest
+    name: Update JS Routing
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.4'
+          extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv, simplexml
+
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+
+      - name: Install Composer dependencies
+        run: composer install --prefer-dist
+
+      - name: Dump JS Routing file
+        run: php bin/console fos:js-routing:dump --format=json --target=admin-dev/themes/new-theme/js/fos_js_routes.json
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+            commit-message: Updated JS Routing file
+            title: 'Updated JS Routing file'
+            body: >
+                    | Questions         | Answers
+                    | ----------------- | -------------------------------------------------------
```

Comment:
Maybe it's not important to fill this template. The template aims to carry data inside the head of the author of the PR to maintainers, in order to help reviewing the PR. This one will be straightforward and "author" cannot answer if there are feedbacks.

Also it means everytime we update the PR template we have to update this GitHub Action